infrastructure/Program.fs

```diff
@@ -15,14 +15,15 @@ let rg = $"{baseName}-rg"
 let myWeb = webApp {
     name webAppName
     system_identity
-    setting "storage-account-name" storageAccountName
+    operating_system OS.Windows
+    setting "StorageAccountName" storageAccountName
     zip_deploy "../publish"
     run_from_package
 }
 
 let storage = storageAccount {
     name storageAccountName
-    add_blob_container "data"
+    add_private_container "data"
```

Comment:
This was the ARM error: `\"code\": \"PublicAccessNotPermitted\",\r\n    \"message\": \"Public access is not permitted on this storage account`.

Switching to a private container does indeed fix it.

Reply:
Note: This was because the storage account itself has blob anonymous access disabled. Perhaps a default changed in Famer or ARM. In any case, it makes more sense for this demo for it to be a private container.